hanko-js/src/lib/HankoClient.ts

```diff
@@ -327,6 +330,18 @@ class WebauthnClient extends AbstractClient {
           reject(e);
         })
         .then((challenge: CredentialRequestOptionsJSON) => {
+          if (
```

Comment:
Would be nice to have a comment here why we need to do this.

Reply:
Thanks, I added a comment.